crates/matrix-sdk/src/event_cache/mod.rs

```diff
@@ -704,7 +704,7 @@ pub enum RoomEventCacheUpdate {
     /// The fully read marker has moved to a different event.
     ReadMarker {
         /// Event at which the read marker is now pointing.
-        event_id: OwnedEventId,
+        move_to: OwnedEventId,
```

Comment:
Sorry, but not sure I like this change either; maybe the read marker was already on that event in which case nothing moves. It's really "at which event id is the read marker pointing to right now".

Reply:
The comment literally says “The fully read marker has moved to a different event”, I was assuming this renaming was alright.

I've renamed to `MoveReadMarkerTo::event_id`.